cmd/config/config_yaml.go

```diff
@@ -512,6 +513,7 @@ func (c *YAMLConfig) parseSchemaSnapshotConfig() (snapshotbuilder.SchemaSnapshot
 			streamSchemaCfg.DumpRestore.IncludeGlobalDBObjects = schemaSnapshotCfg.PgDumpPgRestore.IncludeGlobalDBObjects
 			streamSchemaCfg.DumpRestore.CreateTargetDB = schemaSnapshotCfg.PgDumpPgRestore.CreateTargetDB
 			streamSchemaCfg.DumpRestore.Role = schemaSnapshotCfg.PgDumpPgRestore.Role
+			streamSchemaCfg.DumpRestore.RolesSnapshotMode = schemaSnapshotCfg.PgDumpPgRestore.RolesSnapshotMode
```

Comment:
Maybe we can add a validation for the role snapshot mode, so that only valid values are accepted, rather than setting anything and applying the default silently.